lua/nvim-tree/diagnostics.lua

```diff
@@ -32,7 +32,7 @@ local function add_sign(linenr, severity)
     return
   end
   local sign_name = sign_names[severity][1]
-  vim.fn.sign_place(1, GROUP, sign_name, buf, { lnum = linenr + 1 })
+  vim.fn.sign_place(1, GROUP, sign_name, buf, { lnum = linenr + 1, priority = 2 })
```

Comment:
this will be higher priority than git signs

Reply:
Tested OK.

We should document that behaviour.